services/QuillLMS/db/migrate/20240610155753_create_translation_mappings.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateTranslationMappings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :translation_mappings do |t|
+      t.integer :english_text_id, null: false
+      t.string :source_type, null: false
+      t.integer :source_id, null: false
+      t.string :source_key
```

Comment:
What is source_key?

Reply:
It's not used yet, so I could remove it. The idea is that hints have 3 fields that need to be translated and the key would tell you which field the translation mapped to. In the process of defining this we realized that two of the fields were a little more complicated to translate (most of it would be english, except for a small part) so I made this PR just for translating the one.  I will remove it for now.